content/changes.md

```diff
@@ -8,6 +8,46 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## January, 2024 
+
+### Features 
+ 
+- **Learnosity Partnership Integration** 
+
+    [Learnosity](https://learnosity.com/) has officially announced our partnership and integration into their platform to their custom base, via website, newsletter, and socials! 
+
+    This integration allows us to expand our reach into the L&D space, acting as an important add-on component within Learnosity's deep offering across many different learning domains. We've closed our first partner sale via iCEV, and already have other potential deals beginning to surface.       We will be presenting internally to their sales team later this month to further deepen our partnership and enable their sales team to bring more business to us.  
```

Comment:
This note feels a bit like an internal note to our own staff. The purpose of the Product Updates is for us to communicate relevant platform changes to Qualified's non-Andela customers.

I'm not sure if the Learnosity integration would be relevant to any of our standard customers, unless they're potential customers of Learnosity--if so, maybe it's worth mentioning with a focus on how they can use it. But details about landing iCEV and us pursuing new customers seems outside the purview of Product Updates.

So I'd suggest sticking to:
- bug fixes (e.g. crashes, performance problems, etc)
- new challenges
- new platform features
- new language versions
- improvements to existing features
- general integrations that Qualified customers can use (Greenhouse, Zapier, etc)
- ...in general, technical changes

And avoiding:
- Andela-specific changes (proctoring, resume, etc)
- sales we've closed with customers or integrations that don't affect our clients in general
- marketing announcements (the product changes never mentioned we were acquired by Andela, to pick a somewhat extreme example)
- ...in general, non-technical or internal changes

Reply:
I think it's relevant because like the case of Thinkful who are also Learnosity customers they want to learn more about Learnosity <> Qualified integration. I think the note is promoting and putting the emphasis of a partnership we were able to establish with give us and our customer extra benefits.

Reply:
The topic is fine to keep, but I'd suggest using the [SmartIDE launch](https://docs.qualified.io/changes/#may-11-2022) as a template, which is probably the most marketing-style announcement in the changelog.

The purpose of this page is to provide a list of technical changes to document the evolution of the application/platform itself over time, not acquisitions of new customers or internal meetings--I don't see anything like that in the rest of the changelog. We do not mention any customers by name, for example.

How about something like:

> We've partnered with [Learnosity](https://learnosity.com/) to integrate Qualified coding challenges into Learnosity assessments! Our custom question type enables students to solve project and classic coding challenges in a Learnosity assessment, powered by Qualified's [Embed](https://www.qualified.io/embedded) engine.
> 
> Visit the [release announcement](https://help.learnosity.com/hc/en-us/articles/14870489871133-2024-1-LTS-Release-Announcement) and [Learnosity's Qualified partner page](https://learnosity.com/company/partners/partners-directory/qualified/) to learn more.

This gives them all of the information they need and provides the right level of technical detail, while directing them to the main marketing-style material, and avoiding mentioning iCEV and internal meetings.

Reply:
Maybe Learnosity's [product changelog](https://reference.learnosity.com/releaselogs) would be useful to help clarify the flavor of a typical product changelog (they didn't mention the Qualified integration in the changelog--only in the marketing release notes).

That said, other changelogs are a bit different, so it's not the only way to do it. Some release logs are a bit more "linguistic", like ours, so we don't have to be _quite_ as stark as Learnosity. The technical customer audience, purpose and tone is the same though.